mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -96,6 +96,7 @@ export const EditableValue = <T extends string | number>(props: EditableTextProp
       value={value}
       autoFocus={true}
       onChange={updateValue}
+      style={{width: String(value).length + "ch"}}
     />
```

Comment:
What do you think about to set the width to always be the same (equal to 4 characters, but in pixels)?

Reply:
Done